src/embedded/embedded.h

```diff
@@ -13,7 +13,7 @@
 #ifndef _EMBEDDED_LIB_
 #define _EMBEDDED_LIB_
 
-#include <stddef.h> /* only for size_t */
+#include "monetdb_config.h"
```

Comment:
Nope. I don't want any MonetDB headers in embedded.h.

Reply:
It was for the Windows compilation. I will try to get rid of it.